ci-operator/step-registry/openshift/upgrade/vsphere/loki/openshift-upgrade-vsphere-loki-workflow.yaml

```diff
@@ -0,0 +1,21 @@
+workflow:
+  as: openshift-upgrade-vsphere-loki
+  steps:
+    pre:
+    - ref: ipi-install-monitoringpvc
+    - ref: ipi-install-loki
+    - chain: ipi-vsphere-pre-stableinitial
+    test:
+    - ref: openshift-e2e-test
+    post:
+    - ref: gather-loki
+    - chain: ipi-vsphere-post
+    dependencies:
+      OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: "release:initial"
+      OPENSHIFT_UPGRADE_RELEASE_IMAGE_OVERRIDE: "release:latest"
+    env:
+      TEST_COMMAND: "run-upgrade"
+      TEST_SUITE: "all"
+      TEST_OPTIONS: "abort-at=100"
```

Comment:
Are A->B->A tests what you want by default?  See also #12544.

Reply:
Nope, I was confused when I was watching the progress, "why is it going back to 4.5.x"